src/frontend/get-jira-configuration.ts

```diff
@@ -4,10 +4,25 @@ import { Subscription } from "../models";
 import { NextFunction, Request, Response } from "express";
 import statsd from "../config/statsd";
 import { metricSyncStatus } from "../config/metric-names";
+import { getLogger } from "../config/logger";
+import * as Sentry from "@sentry/node";
+
+const logger = getLogger("get.jira.configuration");
 
 const syncStatus = (syncStatus) =>
 	syncStatus === "ACTIVE" ? "IN PROGRESS" : syncStatus;
 
+const sendFailedStatusMetrics = (installationId: string): void => {
+	const syncError = "No updates in the last 15 minutes"
+	logger.warn(syncError, `Sync failed: installationId=${installationId}`);
+
+	Sentry.setExtra("Installation FAILED", syncError);
+	Sentry.captureException(syncError);
+
+	const tags = { errorMsg: syncError };
+	statsd.increment(metricSyncStatus.failed, tags);
```

Comment:
Just a bit of trivia on metrics: we're not supposed to create metrics with tags that can have a large number of different values (like an error message). The reason is that SignalFX creates a time series for each of the values in that tag. However, given that this is error case doesn't happen too often (hopefully), I think this use case is totally fine, because it won't generate much data.